### PR TITLE
Added Avatar in student section

### PR DESCRIPTION
**Pull request by shreeshjha**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://shreeshjha.github.io/gci18.fossasia.org/)

- Fixes #<Issue Number>

## Description / Changes
I've added my avatar(Ai2.png) in student section as per the the Task states with my Github link as well.

### Before:
screenshot from 2018-11-04 17-32-40

### After: 
screenshot from 2018-11-04 17-09-48

- - - - - - - - - - - -

### CODE REVIEW

**Comment by shreeshjha** [comment]
I've added my avatar in fossasia GCI page as well as my link(github) as well.

**Comment by shreeshjha** [comment]
I've corrected the following issues for merge request

**Comment by fragm3** [comment]
@shreeshjha you have deleted _config.yml, undo this. No need to change config.yml

File: _data/students.yml [hunk 1]
```diff
@@ -73,4 +73,9 @@
   
 - name: Srihari Unnikrishnan
   github: pythongiant
-  image: sr.png
\ No newline at end of file
+  image: sr.png
+  
+- name: Shreesh Kumar Jha
+  github: shreeshjha
+  image: Ai2.png
```
Comment (jogendra) [inline _data/students.yml]: rename your image to your github username like `shreeshjha`.

Comment (shreeshjha) [reply _data/students.yml]: I've created the config file and renamed my img as shreeshjha as well

File: _config.yml [hunk 2]
```diff
@@ -1 +1,3 @@
 baseurl: ""
+
+theme: jekyll-theme-cayman
```
Comment (jogendra) [inline _config.yml]: Do not modify this file. remove this line.

Comment (shreeshjha) [reply _config.yml]: which line i've to remove

Comment (shreeshjha) [reply _config.yml]: as a whole

Comment (jogendra) [reply _config.yml]: the `theme: jekyll-theme-cayman` line

Comment (jogendra) [reply _config.yml]: and squash your commits.

Comment (shreeshjha) [reply _config.yml]: ok

Comment (shreeshjha) [reply _config.yml]: done

**Pull request merged**